homeassistant/components/light/lutron_caseta.py

```diff
@@ -0,0 +1,105 @@
+"""Support for Lutron Caseta lights."""
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, Light)
+from homeassistant.components.light.lutron import to_hass_level
+from homeassistant.components.light.lutron import to_lutron_level
+
+REQUIREMENTS = ['https://github.com/gurumitts/'
+                'pylutron-caseta/archive/v0.2.0.zip#'
+                'pylutron-caseta==v0.2.0', 'paramiko==2.1.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+
```

Comment:
Okay we stay this PR until switch support is done. I merge also the other lutron PR with a pronise that will be have more platform and they are never coming. If you want merge it, let's doit as platform. You can it merge back to component if you have support more platforms